docs/latest/user/quickstart.md

```diff
@@ -2,41 +2,8 @@
 
 This guide will help you get started with Envoy Gateway in a few simple steps.
 
```

Comment:
I vote to keep this info here, you might lose the user if they reach the `Quickstart` page (at the beginning of their EG journey) and might need to click another link to install EG and come back to this page

Reply:
Ok, you mean we should keep all the info related to installation in quickstart page as well.So there will be no changes in quickstart page right ?

Reply:
yah